src/js/popup.js

```diff
@@ -84,6 +84,20 @@ function showNagMaybe() {
 
   function _showError(error_text) {
     $('#instruction-text').hide();
+    $('#error-message').text(error_text);
+    // if error is the 'unexpected error occurred' Firefox issue commonly associated with low disk space, show special message
+    if (typeof browser == "object" && typeof browser.runtime.getBrowserInfo == "function" && error_text.includes("unexpected")) {
+      browser.runtime.getBrowserInfo().then(function (info) {
+        if (info.name == "Firefox") {
+          // fetch the "learn more" message and populate with our FAQ link
+          let learn_more_link = chrome.i18n.getMessage("learn_more_link", ["<a href='https://www.eff.org/privacybadger#faq-I-found-a-bug!-What-do-I-do-now?'>our FAQ</a>"]);
```

Comment:
"our FAQ" is hardcoded in English. Let's bypass adding another message by using "privacybadger.org" here.